addons/mail/static/src/discuss/core/public_web/discuss_sidebar_categories.xml

```diff
@@ -13,7 +13,7 @@
             <input class="form-control rounded-3 border-0 bg-inherit" t-att-class="{ 'lh-1': !ui.isSmall }" disabled="true" placeholder="Find or start a conversation" tabindex="0"/>
             <div class="o-mail-DiscussSidebarCategories-searchClickable position-absolute z-1 opacity-0 cursor-pointer w-100 h-100"/>
         </div>
-        <t t-foreach="store.discuss.allCategories" t-as="cat" t-key="cat_index">
+        <t t-foreach="store.discuss.allCategories" t-as="cat" t-key="cat.id">
```

Comment:
If I'm not mistaken, this could be an issue in 17.4, since this PR: https://github.com/odoo/odoo/pull/146451